apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/SnifferConfigInitializer.java

```diff
@@ -192,4 +192,11 @@ private static InputStreamReader loadConfig() throws AgentPackageNotFoundExcepti
         }
         throw new ConfigNotFoundException("Failed to load agent.config.");
     }
+
+    /**
+     * return the properties read from DEFAULT_CONFIG_FILE_NAME
+     */
+    public static Properties getProperties(){
```

Comment:
Why open this? This is not an API.

Reply:
Because I find no way to get my plugin propeties.

Reply:
when I want to do some properties in my plugin. I find it hard to do that. So I open the this。

Reply:
No, this is a wrong way. If you want to support properties to inject values, there should be an initial mechanism for plugin properties intialization. 

Such as, having a service in the core, and plugin could initial its config object by a callback register. But, notice, this should be called once.

We should have deep discussion about this.

Reply:
I just want something like the way skywalking doing in the  `org.apache.skywalking.apm.agent.core.conf.Config` and used it in any where.If I can get the properties. it was easy to me.

Reply:
No. This is not right for an open source project. It is reasonable to support the config object in the plugin, But the way you wrote is not right.